docs/examples/_server.py

```diff
@@ -169,19 +172,48 @@
 class MockData(DataServiceBase):
     def __init__(self):
         self.tabular_data_requested = False
-        self.tabular_response = [{"PowerPct": 0, "IsPowered": False}, {"PowerPct": 0, "IsPowered": False}, {"Position": 0}]
+        self.tabular_response = [
+            DataClient.TabularData(
```

Comment:
(q) In general, what is the "data" being collected? Any data the robot is giving? And what is binary data vs tabular data?

Reply:
Great question! The data is currently just all data being collected/stored by the robot in app. I don't think I could give a very good answer on the differences between binary and tabular data. I think tabular is going to be mapped data in a lot of ways (e.g., a mapping of function call to output), while binary data is data that's just stored as bytes (e.g., raw files, or a photo). The data team could definitely give a better answer though!